crux-mir/test/conc_eval/slice/subslice.rs

```diff
@@ -0,0 +1,37 @@
+#[cfg_attr(crux, crux::test)]
+fn crux_test() {
+    let mut xs: [u32; 6] = [1, 2, 3, 4, 5, 6];
+    let ys: &mut [u32] = &mut xs;
+
+    let [_, rest @ ..] = ys else { panic!() };
+    assert_eq!(*rest, [2, 3, 4, 5, 6]);
```

Comment:
Does `slice/subslice.rs` exercise the non-`fromEnd` case, or does that only apply to arrays?

Reply:
It looks like all the examples in this test get represented with `fromEnd == True`, and all the examples in the array test get represented with `fromEnd == False`.